subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -259,6 +259,12 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 		lll_adv_iso->bis_spacing = lll_adv_iso->sub_interval;
 		latency_packing = lll_adv_iso->sub_interval * lll_adv_iso->nse *
 				  lll_adv_iso->num_bis;
+		if (latency_packing > sdu_interval) {
+			ll_rx_link_release(link_cmplt);
+			ll_rx_link_release(link_term);
+
+			return BT_HCI_ERR_INVALID_PARAM;
+		}
```

Comment:
Suggest to add empty line after ending curly bracket:
```suggestion
		}
		
```

Not sure if this is something that is commonly done in the controller, but at least it is in the host. Supposedly this increases readability :)

Reply:
Done